src/main/kotlin/uk/gov/dluhc/notificationsapi/mapper/NotificationApiMapper.kt

```diff
@@ -0,0 +1,22 @@
+package uk.gov.dluhc.notificationsapi.mapper
+
+import org.mapstruct.Mapper
+import org.mapstruct.Mapping
+import uk.gov.dluhc.notificationsapi.dto.NotificationDto
+import uk.gov.dluhc.notificationsapi.models.SentCommunicationResponse
+import java.time.OffsetDateTime
+import java.time.ZoneOffset
+import uk.gov.dluhc.notificationsapi.database.entity.Notification as NotificationEntity
+
+@Mapper(
+    uses = [NotificationTypeMapper::class, CommunicationChannelMapper::class, SourceTypeMapper::class, NotifyDetailsMapper::class],
+    imports = [OffsetDateTime::class, ZoneOffset::class],
+)
+interface NotificationApiMapper {
+
+    fun toNotificationDto(notificationSummary: NotificationEntity): NotificationDto
+
+    @Mapping(target = "subject", source = "notifyDetails.subject")
+    @Mapping(target = "body", source = "notifyDetails.body")
+    fun toSentCommunicationsApi(notificationDto: NotificationDto): SentCommunicationResponse
```

Comment:
appreciate it's the pattern, but is it worth skipping the DTO mappings for this and sourceType in the controller / service? They don't add any value currently, unless you plan on using them for other things in the near future

Alternatively if you think it's worth sticking to the pattern, should the DTO object be only what we need from the database object (subject and body)?

Open to discuss this one!!